File: python_ruby/Function/4.py
input_id = input('아이디를 입력하세요.')

def login(_id):
    members = ['egoing', 'kiddo', 'krsc0707']
    for member in members:
        if member == _id:
            return True
    return False

if login(input_id):
    print("Hi," + input_id)
else:
    print('who are you')

File: python_ruby/Function/4.rb
puts("아이디를 입력하세요.")
input_id = gets.chomp()

def login(_id)
    members = ['egoing','krsc0707','kiddo']
    for member in members do
      if member == _id
        return true
    end
  end
  return false
end

if login(input_id)
  puts("Hi ! " + input_id)
else
  puts('who are you?')
end
